translations/en.json

```diff
@@ -1679,7 +1679,7 @@
                         "name": "Name",
                         "new_zone": "New Zone",
                         "passive": "Passive",
-                        "passive_note": "Passive zones are hidden in the frontend and are not used as location for device trackers. This is usefull if you just want to use it for automations.",
+                        "passive_note": "Passive zones are hidden in the frontend and are not used as location for device trackers. This is useful if you just want to use it for automations.",
```

Comment:
Please only update `src/translations` these are maintained by Localize.

Reply:
Reverted this change.